<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QToolButton" name="inFileTool">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="outExtensionLabel">
        <property name="text">
         <string>Output extension:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="resizeCheckbox">
        <property name="text">
         <string>Enable image resizing</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="infileLabel">
        <property name="text">
         <string>Input file path:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="outFolderTool">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="outFolderText"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="outFolderLabel">
        <property name="text">
         <string>Output folder path:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="outNameLabel">
        <property name="text">
         <string>Output name:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QSpinBox" name="outFrameCount">
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="outExtensionText"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="inFileText"/>
      </item>
      <item row="10" column="0" colspan="3">
       <widget class="QPushButton" name="goButton">
        <property name="text">
         <string>Go!</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="blurCheckbox">
        <property name="text">
         <string>Delete blurry images</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="resizeWidthLabel">
          <property name="text">
           <string>Resize width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="resizeWidth">
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="resizeYLabel">
          <property name="text">
           <string>Resize height:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="resizeHeight">
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="outFrameCountLabel">
        <property name="text">
         <string>Output frame count:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="outNameText"/>
      </item>
      <item row="5" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="blurThresholdLabel">
          <property name="text">
           <string>Threshold (lower = blurrier):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="blurThreshold">
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="value">
           <number>800</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="denoiseCheckbox">
        <property name="text">
         <string>Enable image denoising</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>513</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
